static-site/components/people/avatars.tsx

```diff
@@ -0,0 +1,42 @@
+import { ReactElement, ReactNode } from "react";
+
+import styles from "./avatars.module.scss";
```

Comment:
FYI since we just discussed build failures on a call, it's [failing](https://github.com/nextstrain/nextstrain.org/actions/runs/11391880676/job/31696648454) here because of this line

```
./components/people/avatars.tsx
Module not found: Can't resolve './avatars.module.scss'
```

Reply:
(My [build failures](https://github.com/nextstrain/nextstrain.org/pull/1025#issuecomment-2420582251) are unrelated)

Reply:
thanks, that got cleaned up in the most recent pushes.